ember-phone-input/src/components/phone-input.ts

```diff
@@ -29,10 +43,70 @@ import '../styles/styles.css';
   @public
 */
 
-export default class PhoneInputComponent extends Component {
-  @service phoneInput;
+export interface PhoneInputArgs {
+  country: string;
+  number: string | null;
+  disabled: boolean;
+  required: boolean;
+  autocomplete: string | null;
```

Comment:
Would `autocomplete?: string` work here?

Reply:
I'd say the answer is the same as https://github.com/qonto/ember-phone-input/pull/749/files#r1374385425